features/admin.sms-providers.v1/pages/sms-providers.tsx

```diff
@@ -129,19 +129,23 @@ const SMSProviders: FunctionComponent<SMSProviderPageInterface> = (
     const [ existingSMSProviders, setExistingSMSProviders ] = useState<string[]>([]);
 
     useEffect(() => {
-        if (!isSMSProviderConfigFetchRequestLoading && originalSMSProviderConfig?.length > 0) {
+        if (!isSMSProviderConfigFetchRequestLoading) {
             const existingSMSProviderNames: string[] = [];
 
+            let isChoreoSMSOTPProviderEnabled: boolean = false;
```

Comment:
is it good to have such variable names ?

Reply:
Fixed with [e8c51c7](https://github.com/wso2/identity-apps/pull/6504/commits/e8c51c78ab6961006f6c107d9643de74be53bb44)